docs/modules/ROOT/pages/how-tos/oidc.adoc

```diff
@@ -0,0 +1,148 @@
+= Configure OIDC
+
+There is the option to authenticate to vcluster using OpenID Connect.
+This guide assumes that you have setup a vcluster using this component and you have access to an OIDC provider.
+
+
+== Configure IDP
+
+[NOTE]
+====
+This how-to explains how to configure Keycloak, but other OIDC providers should work similarly.
+====
+
+. Open Keycloak and create an OIDC client as follows:
++
+[source,console]
+----
+Client ID: YOUR_CLIENT_ID
+Redirect URIs:
+  http://localhost:8000
+  http://localhost:18000
```

Comment:
Do we need this redirect URI by default for the vcluster OIDC login setup or only for the Commodore redirect? Would be nice to have callouts documenting what the different redirects are used for (`kubectl oidc-login`, `commodore`, ...)

Reply:
These are the ones for `kubelogin` and I used the same one for `commodore`